### ISSUE TITLE
LeViT-128S without distillation 100 epoch training reproduction on 1 GPU

### ISSUE THREAD

**Issue by ktertikas** [facebookresearch/LeViT#12]
Hello and thanks for the great paper and codebase!
I am trying to replicate the numbers reported in Table 5 of the paper, and specifically the A4 model (without distillation), that is reported to achieve 69.7% top-1 accuracy. Would you have any hints as to how to replicate these numbers only having 1 GPU? Modifying the code and using gradient accumulation techniques to replicate the 256 * 32 = 8192 batch size seems to only reach 63.9% top-1 accuracy.
Are there any other steps / tricks that I might be missing? Thanks!

**Comment by btgraham** [comment]
The ablations were trained with 8 GPUs, not 32. I would train with batch-size==256 regardless of the number of GPUs.
The weight decay might need to be adjusted, but I would start with the default value.

**Comment by ktertikas** [comment]
Thank you for the quick response! Could you please let me know the default parameters that you used for the ablations? (Weight decay, etc)

**Comment by btgraham** [comment]
The defaults are all set in `main.py`, except: set distillation-type to none and epochs to 100.

**Comment by ktertikas** [comment]
Cool, thanks a lot for that!
